### ISSUE TITLE
java.lang.reflect.InvocationTargetException

### ISSUE THREAD

**Issue by L4BORG** [NoCheatPlus/Issues#430]
#### Short description of the issue.
Error on startup.

#### How to reproduce the issue (is it random / always happens / side conditions)?
on startup

#### Have you made any (noteworthy) changes to the default configuration of NCP?
No.

#### Do you run any other special plugins that affect game mechanics next to NC+ (adding skills, machines, adding/changing blocks, other anti cheating)?
Viaversion

#### Provide additional information, such as error/debug/violation logs. (For reproducible issues with moving consider: https://github.com/NoCheatPlus/Docs/wiki/Debugging#on-the-fly-debug-output-for-individual-players) Use GitHub gists or http://pastebin.com/ or similar for huge logs, or attach a (zip-) file here. For image uploads, please prefer a site that allows viewing without JavaScript, you can upload file attachments for issues right here.
```[05:52:13] [Server thread/INFO]: [NoCheatPlus] Enabling NoCheatPlus v3.15.2-SNAPSHOT-sMD5NET-b1089
[05:52:13] [Server thread/INFO]: [NoCheatPlus] McAccess set to: 1.8.4-1.8.8 / Spigot-CB-1.8_R3
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.5 blocks.
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.6.1 blocks.
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.7.2 blocks.
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Added block-info for Minecraft 1.8 blocks.
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Inventory checks: FastConsume is available, disabled InstantEat.
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Adding packet level hooks for ProtocolLib (MC 1.8.8)...
[05:52:13] [Server thread/WARN]: [NoCheatPlus] Could not register packet level hook: MovingFlying
[05:52:13] [Server thread/WARN]: [NoCheatPlus] java.lang.reflect.InvocationTargetException
	sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	fr.neatmonster.nocheatplus.checks.net.protocollib.ProtocolLibComponent.register(ProtocolLibComponent.java:137)
	fr.neatmonster.nocheatplus.checks.net.protocollib.ProtocolLibComponent.register(ProtocolLibComponent.java:123)
	fr.neatmonster.nocheatplus.checks.net.protocollib.ProtocolLibComponent.register(ProtocolLibComponent.java:93)
	fr.neatmonster.nocheatplus.checks.net.protocollib.ProtocolLibComponent.<init>(ProtocolLibComponent.java:72)
	fr.neatmonster.nocheatplus.compat.registry.DefaultComponentFactory.getAvailableComponentsOnEnable(DefaultComponentFactory.java:151)
	fr.neatmonster.nocheatplus.NoCheatPlus.onEnable(NoCheatPlus.java:1014)
	org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321)
	org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:332)
	org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:404)
	org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:359)
	org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:318)
	net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:408)
	net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:372)
	net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:327)
	net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:267)
	net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:563)
	java.lang.Thread.run(Thread.java:745)

[05:52:13] [Server thread/INFO]: [NoCheatPlus] Available (and activated) packet level hooks: UseEntityAdapter | OutgoingPosition | KeepAliveAdapter | SoundDistance | CatchAllAdapter
[05:52:13] [Server thread/INFO]: [NoCheatPlus] Version 3.15.2-SNAPSHOT-sMD5NET-b1089 is enabled.```

**Comment by asofold** [comment]
Which version of ProtocolLib is in use here?

Is this a "clean startup" or a reload (or ...)?

**Comment by L4BORG** [comment]
```>ncp version
[02:25:16 INFO]: ---- Version information ----
[02:25:16 INFO]: #### Server ####
[02:25:16 INFO]: git-TacoSpigot-"65fd35f" ~MC: 1.8.8~
[02:25:16 INFO]:   detected: 1.8.8
[02:25:16 INFO]: #### NoCheatPlus ####
[02:25:16 INFO]: Plugin: 3.15.2-SNAPSHOT-sMD5NET-b1087
[02:25:16 INFO]: MCAccess: 1.8.4-1.8.8 / Spigot-CB-1.8_R3
[02:25:16 INFO]: Features:
[02:25:16 INFO]:   blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8
[02:25:16 INFO]:   checks: FastConsume | Gutenberg | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[02:25:16 INFO]:   defaults: pvpKnockBackVelocity
[02:25:16 INFO]:   packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | CatchAllAdapter
[02:25:16 INFO]: #### Related Plugins ####
[02:25:16 INFO]: ProtocolLib v3.6.5
```

Note: this output of NCP version is after we downgraded NCP (but everything else is same).

**Comment by asofold** [comment]
That's interesting, so the changes after build 1087 making used of the 'confirm telelport' packets should not have caused this.

Otherwise build 1084 would be the build before building against ProtocolLib 4.2.1, if at all that will make a difference.

I'll try to reproduce.

**Comment by asofold** [comment]
Ok :9, you're not downgrading randomly...

at least i don't get any exception on startup (no ViaVersion) with the changes that will be in build 1090. So i'll assume it'll be fixed then.

Edit: build 1090 is available on jenkins.

**Comment by L4BORG** [comment]
I'll test it out, also I'm sticking with old version of ProtocolLib because ViaVersion has some issues with newer versions :( (so updating is not really an option)

**Comment by L4BORG** [comment]
Fixed on b#1091
